### PR TITLE
Added new cart page

### PR DESCRIPTION
**Pull request by prathamatwork**
I have added New Cart Page.

### CODE REVIEW

File: src/main/java/Base/BasePage.java [hunk 1]
```diff
@@ -51,6 +51,8 @@ public WebDriver setUp() {
 
 		System.out.println("Added this line from another developer!");
 
+		System.out.println("Another developer added this code to test PULL request functionality");
+
```
Comment (devendra-june-29) [inline src/main/java/Base/BasePage.java:55]: please review your code again

File: src/main/java/Pages/CartPage.java [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package Pages;
+
+public class CartPage {
+
+    public void cartPageCall(){
```
Comment (devendra-june-29) [inline src/main/java/Pages/CartPage.java:5]: Please add another method

Comment (prathamatwork) [reply src/main/java/Pages/CartPage.java:5]: Okay done

File: src/main/java/Base/BasePage.java (hunk 1 above)
Comment (prathamatwork) [reply src/main/java/Base/BasePage.java:55]: Okay done

**Review by devendra-june-29 — APPROVED** [review]

**Pull request merged**